src/i18n/index.ts

```diff
@@ -0,0 +1,52 @@
+import i18next from 'i18next';
+import {list} from '../plugin';
+import {EP_PHOVEA_CORE_LOCALE, ILocaleEPDesc} from '../extensions';
+
+const DEFAULT_LANGUAGE = 'en';
+const DEFAULT_NAMESPACE = 'default_namespace';
+
+/**
+ *  Awaits the translation files registered at the EP_PHOVEA_CORE_LOCALE extension point
+ *  Initialize I18next with the translation files
+ */
+export async function initializeI18n() {
```

Comment:
```suggestion
export async function initI18n() {
```

Should we rename the function to `init`, since we are using this abbrevation everywhere?